python/ray/tests/test_placement_group.py

```diff
@@ -23,6 +23,59 @@ def method(self, x):
         return x + 2
 
 
+def test_placement_group_implicit_resource(ray_start_cluster):
+    @ray.remote(num_cpus=0)  # No resource specified for this actor.
+    class Actor(object):
+        def __init__(self):
+            self.n = 0
+
+        def value(self):
+            return self.n
+
+    cluster = ray_start_cluster
+    num_nodes = 5  # Five node in this cluster.
+    for _ in range(num_nodes):
+        cluster.add_node(num_cpus=2)
+    ray.init(address=cluster.address)
+
+    placement_group = ray.util.placement_group(
+        name="name", strategy="PACK", bundles=[{
```

Comment:
Can we actually change the test to run STRICT_SPREAD and make sure 5 actors are all in a different node instead?

Reply:
same to [this reply](https://github.com/ray-project/ray/pull/12152#discussion_r529286961), The ut test which I want to do here is only one bundle, and five actors, five node, all actors(no resource specified, only pg id and pg bundle index specified ) will correctly scheduled to the node which the bundle located.